tests/utils/MementoTest.cpp

```diff
@@ -1,6 +1,28 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 #include "doctest.h"
+#include "Memento.h"
```

Comment:
Also please fix the import here it should be `#include "utils/Memento.h"` not `#include "Memento.h"`